File: GitHubLike/frontend/src/models/organizations/OrganizationDetailViewDto.ts
export interface OrganizationDetailsViewDto {
  organizationName: string;
  organizationUsersList: OrganizationUserDetailViewDto[];
  createdOn: Date;
  isOwner: boolean;
  isAdmin: boolean;
}

export interface OrganizationUserDetailViewDto {
  userId: number;
  userName: string;
  userRole: string;
}

File: GitHubLike/Modules/OrganizationModule/Models/OrganizationDetailViewDto.cs
namespace GitHubLike.Modules.OrganizationModule.Models
{
    public class OrganizationDetailViewDto
    {
        public string OrganizationName { get; set; }

        public List<OrganizationUserDetailViewDto> OrganizationUsersList { get; set; }

        public DateOnly CreatedOn { get; set; }

        public bool IsOwner { get; set; }

        public bool IsAdmin { get; set; }
    }

    public class OrganizationUserDetailViewDto
    {
        public long UserId { get; set; }

        public string UserName { get; set; }

        public string UserRole { get; set; }
    }
}
